src/unicode/main.zig

```diff
@@ -1,13 +1,238 @@
+const std = @import("std");
 pub const lut = @import("lut.zig");
 
 const grapheme = @import("grapheme.zig");
 const props = @import("props.zig");
 pub const table = props.table;
 pub const Properties = props.Properties;
-pub const getProperties = props.get;
 pub const graphemeBreak = grapheme.graphemeBreak;
 pub const GraphemeBreakState = grapheme.BreakState;
 
 test {
     @import("std").testing.refAllDecls(@This());
 }
+
+/// Build Ghostty with `zig build -Doptimize=ReleaseFast -Demit-unicode-test`.
+///
+/// Usage: ./zig-out/bin/unicode-test [width|class|break|all] [old|zg|ziglyph|all]
+///
+///     width:    this verifies the table codepoint widths match
+///     class:    this verifies the table grapheme boundary classes match
+///     break:    this will verify the grapheme break implementation. This
+///               iterates over billions of codepoints so it is SLOW.
+///
+///     old:      compare against old implementation
+///     zg:       compare against zg
+///     ziglyph:  compare against ziglyph
+///
+/// Note: To disable/enable `old` comparisons, (un)comment sections of these
+/// files (search for "old"):
```

Comment:
I could comment out all the "old" comparison code before merging, if that's preferred. It was useful to verify this switch.